site/class-1-bar/data.tsv

```diff
@@ -0,0 +1,8 @@
+day,hours

```

Comment:
You’re uploading all your files to `site/class-1-bar`, but they should go in `site/class-1-bar/jensorsel`!